package/MDAnalysis/coordinates/base.py

```diff
@@ -757,8 +768,15 @@ class IObase(object):
     units = {'time': None, 'length': None, 'velocity': None}
 
     def convert_pos_from_native(self, x, inplace=True):
-        """In-place conversion of coordinate array x from native units to base units.
+        """Conversion of coordinate array x from native units to base units.
 
+        Keywords
```

Comment:
This is also not the correct numpy-style and we should add `x` in the parameter description.